src/Synapse/Synapse/ChangeLog.md

```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added support for operation of getting droppedsqlpool and geobackup
```

Comment:
The previous changelog is incorrect. All the new items should be put under upcoming release.